mock/environment/__init__.py

```diff
@@ -8,8 +12,36 @@
 
 from common.minimodel import Site, Variant, CloudCover, ImageQuality, WaterVapor
 
+logging.basicConfig(level=logging.INFO)
 
 class Env:
+   
+    def __init__(self):
+        self.site_data = {}
+        self.site_data_by_night = {}
+        self.night = {}
+
+        for site in Site:
+            site_lc = site.name.lower()
+            input_filename = self.data_file_path(f'{site_lc}_weather_data.pickle.bz2')
+
+            if(os.path.exists(input_filename)):
+                logging.info(f'Reading {input_filename}')
+                with bz2.open(input_filename) as input_file:
+                        input_data = pd.read_pickle(input_file)
+                        self.site_data[site_lc] = input_data 
+                        #logging.info(self.site_data[site_lc]["Time_Stamp_UTC"].loc[150:200])
+                        logging.info(f'\t{len(self.site_data[site_lc].columns)} columns, {len(self.site_data[site_lc])} rows')
+                # self.site_data_by_night[site_lc] = self.night
+                # for night in self.site_data[site_lc]["Time_Stamp_UTC"]:
+                #     night_date = night.date()
+                #     #logging.info(night_date)
+                #     self.site_data_by_night[site_lc][night_date] = self.site_data[site_lc][0]
```

Comment:
If you get stuck here and need some help, let me know and I'll see if we can work through the process.